guides/common/modules/proc_creating-a-compliance-policy.adoc

```diff
@@ -1,4 +1,4 @@
-[id="Creating_a_Complicance_Policy_{context}"]
+[id="Creating_a_Compliance_Policy_{context}"]
```

Comment:
Should the old ID stick around so links aren't broken or will that create problems?

Reply:
No, we never keep old IDs around just for the sake for links. We always ensure master is in a working state; which it is according to the GHA pipeline. I also did not find any broken xrefs by hand.

Reply:
I suppose it's OK if you don't cherry pick it. For upstream I'd also say it's safe for now since the documentation is still experimental and we're not too concerned with stability. xrefs are also not too critical since users still end up on the right page. And for Red Hat downstream 3.1 is still considered to be under development so picking to 3.1 is fine.